README.md

```diff
@@ -121,12 +105,11 @@ The judge can also grade in the languages listed below. These languages are less
 * Forth
 * Go
 * Groovy
-* GAS x86/x64/ARM
```

Comment:
Interesting that GAS isn't listed. This is probably an oversight on the judge side.

Reply:
I agree, `GAS x86/x64/ARM` and `NASM x86/x64` seem to be more correct.